class.jetpack.php

```diff
@@ -550,7 +550,9 @@ private function __construct() {
 			add_filter( 'xmlrpc_methods', array( $this, 'remove_non_jetpack_xmlrpc_methods' ), 1000 );
 		}
 
-		if ( defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST && isset( $_GET['for'] ) && 'jetpack' == $_GET['for'] ) {
+		$calypso_env = $this->get_calypso_env();
+
+		if ( ! empty( $calypso_env ) && defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST && isset( $_GET['for'] ) && 'jetpack' == $_GET['for'] ) {
```

Comment:
Added `defined( 'XMLRPC_REQUEST' )` here to fix:
```

Warning: Use of undefined constant XMLRPC_REQUEST - assumed 'XMLRPC_REQUEST' (this will throw an Error in a future version of PHP) in /var/www/html/wp-content/plugins/jetpack/class.jetpack.php on line 555
Stack trace:
1. {main}() /var/www/html/wp-admin/admin-ajax.php:0
2. require_once() /var/www/html/wp-admin/admin-ajax.php:22
3. require_once() /var/www/html/wp-load.php:37
4. require_once() /var/www/html/wp-config.php:84
5. include_once() /var/www/html/wp-settings.php:342
6. require_once() /var/www/html/wp-content/plugins/jetpack/jetpack.php:127
7. Jetpack_Admin::init() /var/www/html/wp-content/plugins/jetpack/class.jetpack-admin.php:284
8. Jetpack_Admin->__construct() /var/www/html/wp-content/plugins/jetpack/class.jetpack-admin.php:23
9. Jetpack::init() /var/www/html/wp-content/plugins/jetpack/class.jetpack-admin.php:34
10. Jetpack->__construct() /var/www/html/wp-content/plugins/jetpack/class.jetpack.php:353
```